tests/upgrades/test_repository.py

```diff
@@ -106,7 +106,7 @@ class TestScenarioCustomRepoCheck:
         1. Before Satellite upgrade.
         2. Create new Organization and Location.
         3. Create Product, custom repo, cv.
-        4. Create activation key and add subscription in it.
+        4. Create activation key.
```

Comment:
So since this is an upgrade test from 6.15 to 6.16, We still need to have some sca related testing in the preupgrade parts of these tests. I left these in for a reason... but they will have to be removed in 6.17